src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -129,7 +129,7 @@ export const SearchResult = ({ school, onSearchResultClick }) => (
         className="vads-u-font-size--h3 vads-u-font-weight--bold vads-u-font-family--serif vads-u-margin--0"
         data-e2e-id="result-title"
       >
-        {deriveNameLabel(school)}
+        <h3 className="vads-u-margin--0">{deriveNameLabel(school)}</h3>
```

Comment:
Can we put an h3 inside a dd tag without causing an a11y issue? https://html.spec.whatwg.org/multipage/grouping-content.html#the-dd-element (I genuinely don't know)

Reply:
Neither do I, @joshkimux , do you have anything thoughts?

Reply:
FYI, with voice over, it navigates to the h3's just fine when navigating through headings.